@here/harp-datasource-protocol/lib/ExprEvaluator.ts

```diff
@@ -50,19 +51,66 @@ const operatorDescriptors = new Map<string, OperatorDescriptor>();
  * @hidden
  */
 export class ExprEvaluatorContext {
+    private _partialEvaluation = false;
```

Comment:
It's minor but so far we've used `m_` everywhere for all private members.

Reply:
I renamed it, but in this module we usually prefer `_`.